### DISCUSSION TITLE
Regarding importing neper model in 3DEC 7

### DISCUSSION THREAD

**Discussion by Ananyashree0711** [neperfepx/neper#822] [Q&A] [answered]
Dear All,
I have generated a model in NEPER and I am trying to import it in 3DEC 7 by using "-format 3dec " at the end. This worked fine in 3DEC 5.2 but isnt compatible in 3DEC 7.
Kindly suggest a solution for this.

**Comment by rquey** [comment, marked as answer]
I have no solution. You'll have to find out what 3DEC 7 needs.

**Comment by lsspikaqiu** [comment]
Hello, in 3dec, you can convert the ".3dec "file format generated by NEPER into 3dec7.0 version of the command, you can successfully run
